stories/bar/bar.stories.js

```diff
@@ -44,7 +44,7 @@ export const Default = () => `
 <div class="fd-bar">
     <div class="fd-bar__left">
         <div class="fd-bar__element">
-            <span class="fd-bar__element fd-bar__element--title" aria-label="text">TEXT</span>
+            <span class="fd-bar__element--title" aria-label="text">TEXT</span>
```

Comment:
You can't have modifier class `fd-bar__element--title` without base class `fd-bar__element` on the same level

Reply:
Instead of:
```
<div class="fd-bar__element">
            <span class="fd-bar__element--title" aria-label="text">TEXT</span>
</div>
```

You need:
```
<div class="fd-bar__element fd-bar__element--title">
          <h4 class="fd-title fd-title--h4">Title</h4>
</div>
```

Reply:
Hi @InnaAtanasova ,

  Added `fd-bar--title` ( instead of` fd-title`,` fd-title--h4` and `fd-bar__element--title` 3 classes) as shown below.

```
<div class="fd-bar__element">
          <h4 class="fd-bar--title">Title</h4>
</div>
```

Please do let me know, if this approrach is ok..will update the whole PR details accordingly .


Thank You